aiomcache/client.py

```diff
@@ -61,6 +64,18 @@ def __init__(self, host: str, port: int = 11211, *,
         :param port: memcached port
         :param pool_size: max connection pool size
         :param pool_minsize: min connection pool size
+        :param ssl: whether to use TLS against the memcache server. If ssl
+            is a ssl.SSLContext object, this context is used to create the
+            transport; if ssl is True, a default context returned from
+            ssl.create_default_context() is used. This parameter is sent
+            through to asyncio.create_connection(), so see
+            https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.create_connection
+            for a detailed description.
+        :param ssl_handshake_timeout: time in seconds to wait for the TLS
+            handshake to complete, if using TLS. This parameter is sent
+            through to asyncio.create_connection(), so see
+            https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.create_connection
+            for a detailed description.
```

Comment:
Looking at this once more, I'm just wondering if it would be useful to pass arbitrary arguments to open_connection(). e.g. with a `conn_args: Dict[str, Any]` or similar.

Reply:
Yeah, that's probably a good idea. That will e.g. allow Python 3.11 users to set the `ssl_shutdown_timeout` parameter. I'll change it.